values.yaml

```diff
@@ -33,6 +38,12 @@ common:
     # "gp2" would be appropriate.
     class: "standard"
 
+# Create custom config file
+# You will need to tweak `command` & `args` to use it instead of default file
+# WARNING: this can broke install/upgrade so use only if you know what are you doing
+configmap:
```

Comment:
Maybe adding a configmap override option here would also be a good idea for people wanting to use their own configmaps.

Reply:
that is the idea, if you check the sample configmap I provided, it make uses of same env vars currently used in this chart, once the chart is installed, you can modify it as much as you wish